src/components/TodoItem.js

```diff
@@ -1,18 +1,29 @@
 import React, { Component } from "react";
+import styles from "./TodoItem.module.css";
 
 class TodoItem extends Component {
   render() {
+    const completedStyle = {
+      fontStyle: "italic",
+      color: "#595959",
+      opacity: 0.4,
+      textDecoration: "line-through",
+    };

```

Comment:
It would be better to use a CSS file for the styling so your JS file ca work more faster